src/en-us.strings.js

```diff
@@ -190,5 +190,18 @@ export default {
 	'pdfReader.convertToHighlight': 'Convert to Highlight',
 	'pdfReader.convertToUnderline': 'Convert to Underline',
 	'pdfReader.size': 'Size',
-	'pdfReader.merge': 'Merge'
+	'pdfReader.merge': 'Merge',
+    'pdfReader.a11yAnnotationModifierMac': 'Option',
+    'pdfReader.a11yAnnotationModifier': 'Alt',
+    'pdfReader.a11yAnnotationModifierControl': 'Control',
+    'pdfReader.a11yTextualAnnotationFindInDocumentInstruction': 'To turn a search result into a highlight or underline annotation, press',
+    'pdfReader.a11yTextualAnnotationInstruction': 'To annotate text via the keyboard, first use “Find in Document” to locate the phrase. Then, to turn the search result into an annotation, press Control -',
```

Comment:
Are we able to use placeholders in these strings? Because this could be a little more concise as `To annotate text via the keyboard, first use “Find in Document” to locate the phrase, and then press Control - {whatever} to turn the search result into an annotation.`

Reply:
I attempted some time ago and ran into issues trying to get placeholders to work. I'll give it another shot

Reply:
On a separate note, just to make sure I'm not mistaken, these strings do not actually get used when the desktop app is built, right? I have to add these entries to zotero.properties for these new `pdfReader.*` values to get announced.

Reply:
Correct. Otherwise I think the reader will fall back to the English strings (and possibly use placeholders for non-English?).